### PR TITLE
fix(tauri.js) beforeDevCommand race condition

### PR DESCRIPTION
**Pull request by lucasfernog**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `latest` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

It has been reported on Dicord when using a Quasar app.

### CODE REVIEW

File: cli/tauri.js/src/runner.ts [hunk 1]
```diff
@@ -67,6 +68,18 @@ class Runner {
       ls.stderr?.pipe(process.stderr)
       ls.stdout?.pipe(process.stdout)
       this.beforeDevProcess = ls
+
+      let devTryCount = 0
+      const devTryTimeout = 3000
+      while (!(await isReachable(devPath))) {
+        log('Waiting for your dev server to start...')
+        await new Promise(resolve => setTimeout(resolve, devTryTimeout))
```
Comment (nothingismagick) [inline cli/tauri.js/src/runner.ts:76]: should we use `setInterval` instead? sometimes devservers take forever to spin up...

Comment (lucasfernog) [reply cli/tauri.js/src/runner.ts:76]: more than 15s?

Comment (nothingismagick) [reply cli/tauri.js/src/runner.ts:76]: if you are running 10 electron apps and 100 chrome tabs with only 8gb you might swap and i've had to wait 60 seconds sometimes

Comment (lucasfernog) [reply cli/tauri.js/src/runner.ts:76]: let's just run it forever then?

**Review by nothingismagick — APPROVED** [review]

**Pull request merged**